src/Console/FlushCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Laravel\Horizon\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Redis\Factory as RedisFactory;
+
+class FlushCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'horizon:flush';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = "Truncate all horizon-related information from Redis";
+  
+  /**
+   * Execute the console command.
+   * @param RedisFactory $redis
+   * @return void
+   * @throws \RedisException
+   */
+    public function handle(RedisFactory $redis)
+    {
+      if(app()->environment('local')) {
+        $redis->connection('horizon')->client()->flushAll();
+        $this->info('All queue jobs flushed successfully.');
+      }
```

Comment:
Why do we need to check for `local` environment here?

Reply:
We have special trait to add confirmation whenever we need to run a command on `production` envrionment.

Reply:
@crynobone, I just added the confirmation trait. Do you think we should include a `--force` option? In my view, it's better not to include it, as the action is quite destructive, and requiring confirmation would be safer. What do you think?